test/util/test_case.hpp

```diff
@@ -75,11 +90,34 @@ namespace ngraph
                     m_backend->create_tensor(function_output_type, function_output_shape));
 
                 m_expected_outputs.emplace_back(std::make_shared<ngraph::op::Constant>(
-                    function_output_type, function_output_shape, values));
+                    function_output_type, expected_shape, values));
 
                 ++m_output_index;
             }
 
+            template <typename T>
+            void add_expected_output(const std::vector<T>& values)
+            {
+                auto shape = m_function->get_results().at(m_output_index)->get_shape();
```

Comment:
What if output shape is broken?
Default shape as 1D tensor can be better in this case.
Now expected shape is connected with ngraph which can be invalid.

Reply:
We cannot assume that the output shape will be flat.